app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
I am not going to force it, especially that it is late indeed, but I do not think that these arguments are valid. First, we changed a lot in the database schema so why not this particular name? The reason for this would be to go with the convention which says table names are plural. As for grammar, I think it is rather common practice to make plural forms of acronyms. Also, [wikipedia says it is correct](http://en.wikipedia.org/wiki/Quantitative_trait_locus) ;)